blockbase/sass/base/_header.scss

```diff
@@ -10,3 +10,12 @@
 		text-decoration: none;
 	}
 }
+
+h1, h2, h3, h4, h5, h6 {
+	&.has-background {
+		padding-top: var(--wp--custom--margin--vertical);
```

Comment:
Is there a reason you didn't use the shorthand?
`padding: var(--wp--custom--margin--vertical) var(--wp--custom--margin--horizontal)`

Reply:
Only because I refactored from using four distinct variables and the thought just didn't cross my mind.